Makefile

```diff
@@ -32,3 +34,23 @@ pull_translations:
 	git clean -fdX conf/locale/rtl
 	git clean -fdX conf/locale/eo
 	i18n_tool validate
+
+requirements: ## install development environment requirements
+	pip install -qr requirements/edx/development.txt --exists-action w
+
+upgrade: ## update the pip requirements files to use the latest releases satisfying our constraints
+	pip install -q pip-tools
+	pip-compile --upgrade -o requirements/edx/coverage.txt requirements/edx/coverage.in
+	pip-compile --upgrade -o requirements/edx/paver.txt requirements/edx/paver.in
+	pip-compile --upgrade -o requirements/edx-sandbox/shared.txt requirements/edx-sandbox/shared.in
+	pip-compile --upgrade -o requirements/edx-sandbox/base.txt requirements/edx-sandbox/base.in
+	pip-compile --upgrade -o requirements/edx/base.txt requirements/edx/base.in
+	pip-compile --upgrade -o requirements/edx/testing.txt requirements/edx/testing.in
+	pip-compile --upgrade -o requirements/edx/development.txt requirements/edx/development.in
+	scripts/post-pip-compile.sh requirements/edx/base.txt requirements/edx/coverage.txt \
+	    requirements/edx/development.txt requirements/edx/paver.txt requirements/edx/testing.txt \
+	    requirements/edx-sandbox/shared.in requirements/edx-sandbox/base.in
```

Comment:
Were these `edx-sandbox` files supposed to be the `.txt` versions?

Reply:
Yes, good catch.  Fixing.